src/components/menu/MenuItem.tsx

```diff
@@ -68,7 +68,7 @@ const MenuItem: GenericComponent<MenuItemProps> = ({
   const restProps = omitBoxProps(others);
 
   return (
-    <Box {...boxProps} data-teamleader-ui="menu-item" display="flex" element="li">
+    <Box {...boxProps} data-teamleader-ui="menu-item" display="flex" element="li" tabIndex={-1}>
```

Comment:
I'm not 100% convinced this is the correct solution. I don't think the `li` element needs to be focussable, just the buttons themselves which is already the case.

Is this perhaps an issue that could be solved by having different styling between hovered and focussed menu items?
Or maybe it is possible to make the `Menu` component (or an invisible component before the first menu item if that causes the menu items to be skipped) focussable. That way we keep the correct ux and we also aren't forced to focus the first button by default.

Reply:
For the first suggestion, wasn't able to find a way with different styling for the menu items. But I could definitely try the second approach, and figure out a way if possible 👍.

Reply:
Adding an invisible item before the other menu items, it does the trick. Will update the PR with the changes now. Thank you for the remark.